packages/api-server/src/lib/data-extractors.ts

```diff
@@ -3,14 +3,29 @@ import { ServerResponse, IncomingMessage } from "http";
 import { Readable, Writable } from "stream";
 import { CeroError } from "./definitions";
 
+/**
+ * Returns result of the function call if object is a function or the object itself.
+ *
+ * @param {any} object Object.
+ * @param {IncomingMessage} req Request.
+ * @returns {Promise<any>} Promise resolving to the result of the function call or to the object itself.
+ */
 export async function getObject(object: any, req: IncomingMessage): Promise<any> {
     if (typeof object === "function") {
-        return await object(req);
+        return object(req);
     }
 
     return object;
 }
 
+/**
+ * Returns result of the function call if object is a function or the object itself.
+ *
+ * @param {any} object Object.
+ * @param {IncomingMessage} req Request.
+ * @param {ServerResponse} res Response.
+ * @returns {Promise<any>} Promise resolving to the result of the function call or to the object itself.
+ */
```

Comment:
It bothers me a bit since this function is almost identical to the one above (apart from additional param and return type) - and since it still accepts `any` as first param the possible `results` would be the same as in `getObject()`? And `getWritable` name seems to be somehow inconsistent with docs / what this function does.

I know that it is probably used in a different context, but without it, it's quite confusing. And I know that this PR is about docs, not code :see_no_evil: